capture/devconfig.py

```diff
@@ -1,28 +1,24 @@
-# devconfig.py
-
 import platform
 
 #######################################
-
-#  version control
+# version control todo: ian where does this get used? Not sure if we should have to manually do this
```

Comment:
This comment would suggest that there might be a better way to keep track of the version.  I am open to suggestions and we should implement something for V3.  I don't know if it is needed for V2.